### PR TITLE
Added winter Delphi conferences

### PR DESCRIPTION
**Pull request by marcobreveglieri**
Added these new conferences to the list:

+ ITDevCon European Delphi Conference 2024 (Rome)
+ Delphi Day Italian Conference 2024 (Padova)

### CODE REVIEW

**Review by cortinico — APPROVED** [review]

**Comment by cortinico** [comment]
Grazie mille @marcobreveglieri

**Comment by cortinico** [comment]
@marcobreveglieri this PR actually broke the CI on main. Could you check why that happened?

**Comment by marcobreveglieri** [comment]
@cortinico I've just added two new Markdown files: how could they have broken the build?
Sorry, I am not a Ruby expert. I've tried to run the build myself locally but I am not able to reproduce the error. :|

File: _conferences/delphi-day-italian-conference-2024-padova.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: "Delphi Day Italian Conference - Padova 2024"
+website: https://www.delphiday.it/
+location: Padova, Italy
+online: false
+
+date_start: 2024-20-11
+date_end: 2024-20-11
```
Comment (cortinico) [inline _conferences/delphi-day-italian-conference-2024-padova.md:8]: Those 2 dates were invalid and causing the build to fail @marcobreveglieri 
I've fixed it on main already

Comment (marcobreveglieri) [reply _conferences/delphi-day-italian-conference-2024-padova.md:8]: I checked it visually several times but still couldn't see the problem, even though it was right in front of my eyes! :)

Thanks a lot for your support (and patience)! ;)

**Pull request merged**